gf-webapp/src/component/pageBorder/header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import {useEffect} from "react";
+import {NavLink} from "react-router-dom";
+import {useLocation} from "react-router-dom";
+import {useDictionary} from "src/logic/DictionaryContext/useDictionary";
+import {useScrollPosition} from "src/utils/domEventsUtils/useScrollPosition";
+import {Burger} from "src/component/pageBorder/header/burger/Burger";
+import {Contacts} from "src/component/pageBorder/header/contacts/Contacts";
+import {LanguagesList} from "src/component/pageBorder/header/languagesList/LanguagesList";
+import styles from "src/component/pageBorder/header/Header.module.scss";
+
+export const Header = () => {
+  const dictionary = useDictionary().dictionary;
+
+  const scrollPosition = useScrollPosition();
+
+  const location = useLocation();
```

Comment:
Delete empty lines between const declarations. You can use LanguagesList.tsx as an example

Reply:
Done